modules/distr-tracing-rn-technology-preview.adoc

```diff
@@ -35,7 +35,7 @@ For services that have not sent traces within the last 15 minutes, those traces
 
 Expanded support for the Tempo Operator is planned for future releases of {DTProductName}.
 Possible additional features might include support for TLS authentication, multitenancy, and multiple clusters.
-For more information about the Tempo Operator, see link:https://tempo-operator.netlify.app[the Tempo community documentation].
+For more information about the Tempo Operator, see link:https://grafana.com/docs/tempo/latest/setup/operator/[the Tempo community documentation].
```

Comment:
If I understand correctly, it's (as a matter of fact) the documentation for `Community Tempo Operator`. If so, then I suggest changing it a bit:
```suggestion
For more information about the Tempo Operator, see the documentation for the link:https://grafana.com/docs/tempo/latest/setup/operator/[Community Tempo Operator].
```
![image](https://github.com/openshift/openshift-docs/assets/78564304/059404e3-9d83-49de-b29c-21073fbf45c2)

Reply:
Change applied